node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentContextImpl.java

```diff
@@ -296,8 +300,9 @@ public String getCloudNativeRegionName() {
                             return getId().region().value();
                         }
                     }),
-                    Optional.ofNullable(pathToContainerStorage).orElseGet(() -> fileSystem.getPath("/home/docker")),
-                    Optional.ofNullable(pathToVespaHome).orElseGet(() -> fileSystem.getPath("/opt/vespa")),
+                    fileSystem,
+                    fileSystem.getPath("/home/docker/container-storage"),
```

Comment:
The `container-storage` addition means more typing in tests... :(

Reply:
yes, but it matches the expected path.  I was hoping this could lead to simplifying the NodeAgentContextImpl constructor by removing 2 path parameters - but I ended up keeping them.